app/controllers/users_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
       session[:user_id] = user.id
       redirect_to dashboard_user_path(user)
     else
-      flash[:errors] = user.errors.full_messages
+      flash[:notice] = 'Please fill in both an email and a password'
       flash[:notice] = "Passwords do not match" if !matching_password?
```

Comment:
I'm not sure this is the best idea, this would leave out errors like already in database.

Reply:
No errors were displaying while I was adjusting the tests for the nav bar, but I completely agree! We should look into this together.